build-system/tasks/extension-helpers.js

```diff
@@ -386,7 +386,7 @@ function buildExtension(
   }
 
   // minify and copy vendor configs for amp-analytics component
-  if (name === 'amp-analytics' && argv.compile_vendor_configs) {
+  if (name === 'amp-analytics') {
```

Comment:
@jonathantyng-amp could you please confirm that the build works even if `gulp generate-vendor-jsons` hasn't been run (no such json files to copy). In that case what will happen? Will the analytics-vendors folder be created? Thanks.

Reply:
Tested locally and confirmed that build works with no JSON files to copy. If there are no vendor JSONs, `dist/analytics-vendors` folder will not be created

Reply:
Great! Thanks